conductr_cli/resolvers/uri_resolver.py

```diff
@@ -20,8 +20,15 @@ def resolve_file(cache_dir, uri, auth=None):
         os.makedirs(cache_dir, mode=0o700)
 
     try:
+        file_protocol = 'file://'
         file_name, file_url = get_url(uri)
 
+        if file_url.startswith(file_protocol):
+            file_path = file_url[len(file_protocol):]
+
+            if os.path.exists(file_path):
+                return True, file_name, file_path
+
```

Comment:
@longshorej - I think this should be ok.

This would mean the local file won't be cached, but I think there should not be any other impact.